### DISCUSSION TITLE
Any custom brushes y’all recommend?

### DISCUSSION THREAD

**Discussion by 6xolotl** [keiwando/toonsquid#318] [General]
Title says it all. I wanna expand my brush library, but I can’t figure out making them and I haven’t had any luck finding brushes online. What about you guys?

**Comment by trinhpxl** [comment]
i have a bunch of friends who use TS bc i hardcore advertised it LMFAOO so heres some brushes they made !! i’ve been collecting them :3 the only brush by me is the marker one ! https://drive.google.com/drive/folders/1-0_QgHIKiqT4xOqc8LLF66jy4PCQCxRb

(lmk if link doesnt work! )

**Reply by 6xolotl** [reply]
YOU ARE A LIFESAVER!!!! starred the drive!!! :DDD

**Reply by trinhpxl** [reply]
ofc ^^ so glad u do, enjoy ! 
i suggest if any1 also has custom brushes they r collecting, to share as well :D !

**Reply by DWrightAnimation** [reply]
Could you make a ToonSquid brush that looks like a crayon, in case I needed to animate something that looks like it was drawn by a child?

**Reply by snorlings** [reply]
I know this is like super late but is there a way you could repost these files? I can’t import any of them into ToonSquid for whatever reason but I’m in LOVE with them

**Reply by Cat-Butler** [reply]
Hi @snorlings  

Existing link works fine here, should not need a repost. If you use the link given, you should see a list of folders in your browser (or possibly in the Google Drive app), each containing a .tsbrush (single brush) or .tsbrushes (set of brushes)  file and a preview image.

Tap to open/enter a folder. Download the .tsbrush(es) file (this can vary a bit between browsers, so I'm not going to be specific. Look for  a download/down-arrow icon, especially after tapping to select the file). If asked, use Save to Files, otherwise it will probably download directly to your Downloads folder (on your iPad).
If you instead end up with a .zip (possibly .tsbrush(es).zip) file, you might need to tap and hold on it until you get a pop-up menu and select the Decompress option, which will unzip it to a separate file/folder in the same location.

In ToonSquid, select the Brush tool then tap it again to open the brush library. Tap the + in the top right corner, select Import, then browse to your downloaded .tsbrush(es) file. Tap to select it, and it should import into ToonSquid in it's own folder in the brush library.

**Reply by stawwe** [reply]
> i have a bunch of friends who use TS bc i hardcore advertised it LMFAOO so heres some brushes they made !! i’ve been collecting them :3 the only brush by me is the marker one ! https://drive.google.com/drive/folders/1-0_QgHIKiqT4xOqc8LLF66jy4PCQCxRb
> 
> (lmk if link doesnt work! )

i'm not allowed to download it for some reason?

**Reply by stawwe** [reply]
when i tried to download my old brushes from the drive, it says i cant downlaod it

**Reply by Cat-Butler** [reply]
Working fine here. 
Using Safari web browser on iPad (other browsers/platforms/apps might behave differently) , after following that link I tapped (always a single tap for web links)  on the HERO PACK folder, then tapped the .tsbrushes file in that folder. 
It showed the internals of the .tsbrushes file (a long string of hex codes), ignore that and tap the download icon (down arrow) to download. Might take a couple of seconds for Google Drive to respond at times.

![brush-download](https://github.com/user-attachments/assets/be16b7b4-df74-49c8-9faa-a099d196a394)

**Reply by SantiAnimates** [reply]
Hi @Cat-Butler !
If this conversation is still on, I’m still trying to import the brushes into ToonSquid but nothing seems to work. I downloaded all the stuff and extracted the files but I can’t import anything. There’s this file with tsbrush on it which I guessed was the brush but it was a .zip file. In it there was a file called “Brushes” with this single file with a bunch of letters and numbers that I guess are supposed to be the brush but when trying to import it ToonSquid doesn’t recognize it. Idk what to do and I REALLY WANT THESE BRUSHES.

**Reply by Cat-Butler** [reply]
Hi @SantiAnimates !

You are down a level too far. You should be seeing files ending in .tsbrush or .tsbrushes - yes, these are zip files internally, but that is how ToonSquid accepts and processes them. If you end up with something.tsbrush.zip (overly helpful web browser or Google Drive trying to tell you the download is a zip file?), rename it back to something.tsbrush - this should now import without problems.

If you are seeing letters and numbers (as a file or folder) you are looking at the internal pieces of the brush file.

**Comment by Cat-Butler** [comment]
A few I've been experimenting with, nothing special: https://drive.google.com/file/d/1HdbwiB9oWN4BcPij7CZSDb6YPR2k37fU/view (should be a download arrow in the upper right).

While ToonSquid can import some .abr (Photoshpo brush) files, many .abr files do not currently import successfully.
If you want some vegetation brushes (backgrounds, gardens, window boxes, pots), I found these imported without problems (well, for me there were no problems - import might take a little time): https://maddybellwoar.gumroad.com/l/ghiblibrush
It's a paid set, but fairly good value. 
Have not yet tried any of [the creator's other brush sets](https://maddybellwoar.gumroad.com/).

**Comment by ota-works** [comment]
Has anyone created any interesting inking brushes for TS? I'd love to clean up frames with brushes with nice textures like a pencil brush or varied textured brush? (one of the reasons why I hope Procreate brushes would be able to be imported perfectly into TS. If so, everything changes, lol. Unfortunately, abr brushes don't load perfectly in TS)

**Reply by bubbapi** [reply]
I really like this set:

https://bbrushes.gumroad.com/l/toonsquid
